ocp_resources/data_science_cluster.py

```diff
@@ -0,0 +1,88 @@
+from ocp_resources.resource import Resource
+
+
+class DataScienceCluster(Resource):
+    """
+    DataScienceCluster Resource for RHOAI operator
+
+    https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2.5/html/installing_and_uninstalling_openshift_ai_self-managed/index
```

Comment:
This is a link to `Installing and uninstalling OpenShift AI Self-Managed`, you need to replace it with link to the resource definition.

Reply:
@myakove I didn't find any direct link to this resource, it is part of installation guide of data science product.

Reply:
please add also `https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/apis/datasciencecluster/v1/datasciencecluster_types.go`
and `https://github.com/opendatahub-io/opendatahub-operator/blob/a6eb73af559f9ba7438c529420f6e55c83b045af/config/crd/bases/datasciencecluster.opendatahub.io_datascienceclusters.yaml`

Reply:
Thanks, will update!